zio-json/shared/src/main/scala/zio/json/decoder.scala

```diff
@@ -676,6 +676,128 @@ private[json] trait DecoderLowPriority3 {
     }
 }
 
+// The code in JavaTimeWorkaround is a more Scala-friendly port of the Duration.parse method from JDK 16 to cope with
+// a bug resulting in incorrect durations in JDK 8 (see https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8054978)
+private[json] object JavaTimeWorkaround {
```

Comment:
Nitpicking: I’d move this to a new file, maybe in an `util` package nested under the package of this class

Reply:
Will make the change!